com.ibm.wala.ide.tests/src/test/resources/wala.testdata.txt

```diff
@@ -0,0 +1,3 @@
+Primordial,Java,stdlib,none
```

Comment:
I wish we didn't have to duplicate this file, but didn't immediately see a better solution

Reply:
The original is `com.ibm.wala.core/src/test/resources/wala.testdata.txt`, right? We should be able to have Gradle copy that original file for us as part of the `:com.ibm.wala.ide.tests:processTestResources` task.

Try removing this manually copied file and instead adding something like the following to `com.ibm.wala.ide.tests/build.gradle`:

```gradle
tasks.named('processTestResources', Copy) {
	from project(':com.ibm.wala.core').sourceSets.test.resources
	include 'wala.testdata.txt'
}
```

Reply:
It worked! 0d3cbea

Reply:
W00t!